web/controllers/days_controller.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Janitor.DaysController do
+  use Janitor.Web, :controller
+  import Janitor.DaysQueryHelpers
+  alias Janitor.Repo
+
+  def index(conn, %{month: month}) do 
+    days = Repo.all(from_month(month))
+    render conn, "days.json", days: days
+  end 
+
+  def index(conn, _) do 
+    days = Repo.all(from_month())
+    render conn, "days.json", days: days
```

Comment:
Why is `"days.json"` a parameter here if there is no other alternative at the moment? Please drop it.